cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -3542,18 +3544,6 @@ macro(resolve_dependency_absl)
                    APPEND
                    PROPERTY INTERFACE_LINK_LIBRARIES ${CoreFoundation})
     endif()
-    set_property(TARGET absl::type_traits PROPERTY INTERFACE_LINK_LIBRARIES absl::config)
```

Comment:
These are duplicated above so I removed them here; I suspect these were an artifact of a bad merge